src/MIDebugEngine/AD7.Impl/AD7Module.cs

```diff
@@ -71,6 +73,17 @@ int IDebugModule2.GetInfo(enum_MODULE_INFO_FIELDS dwFields, MODULE_INFO[] infoAr
                     info.m_dwLoadOrder = this.DebuggedModule.GetLoadOrder();
                     info.dwValidFields |= enum_MODULE_INFO_FIELDS.MIF_LOADORDER;
                 }
+                if ((dwFields & enum_MODULE_INFO_FIELDS.MIF_TIMESTAMP) != 0)
+                {   
+                    long ft = File.GetLastWriteTimeUtc(this.DebuggedModule.SymbolPath).ToFileTime();
```

Comment:
>long [](start = 20, length = 4)

You are also missig a try/catch around this to not return a value if GetLastWriteTimeUtc throws.